### PR TITLE
Correct path in console command

### PR DESCRIPTION
**Pull request by mbabker**

### CODE REVIEW

**Comment by mbabker** [comment]
I signed it!

File: docs/user-guide/downward-api/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Use the [`examples/downward-api/dapi-pod.yaml`](dapi-pod.yaml) file to create a
 downward API.
 
 ```console
-$ kubectl create -f docs/user-guide/downward-api/dapi-pod.yaml
+$ kubectl create -f examples/downward-api/dapi-pod.yaml
```
Comment (thockin) [inline docs/user-guide/downward-api/README.md]: This is not right.  The file exists as https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/downward-api/dapi-pod.yaml

**Comment by thockin** [comment]
I think maybe your git checkout is stale?

**Comment by mbabker** [comment]
Shouldn't be, I just hit the edit button on the repo here when the patch was created.  When browsing the docs at http://kubernetes.io/v1.0/docs/user-guide/downward-api/README.html the example refers to the docs path, but when a release is downloaded the `docs` folder isn't included but all of the referenced examples are in the `examples` folder.  So in a lot of places where I was copy/pasting things, I was getting file not found errors because of that path use.

**Comment by thockin** [comment]
Ahh, yeah. docs were updated after release of the binaries :(  Our bad.

**Pull request closed without merging**